core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -438,6 +438,75 @@ def ensure_user_data_structure_is_in_place() -> bool:
     return False
 
 
+def load_pi5_overlays_in_runtime() -> bool:
+    loaded_overlays = run_command("dtoverlay -l", check=False).stdout.split("\n")
+    loaded_overlays = [overlay.split(": ")[1].strip() for overlay in loaded_overlays if ": " in overlay]
+    loaded_overlay_names = [overlay.split(" ")[0] for overlay in loaded_overlays if "dtparam" not in overlay]
+    loaded_dt_params = [overlay.split(" ")[-1] for overlay in loaded_overlays if "dtparam" in overlay]
+
+    logger.info("Loaded overlays: ")
+    for overlay in loaded_overlays:
+        logger.info(overlay)
+
+    logger.info("Loaded dt params: ")
+    for dt_param in loaded_dt_params:
+        logger.info(dt_param)
+
+    dtparams_to_load = [
+        "i2c_arm=on",
+        "i2c_arm_baudrate=1000000",
+    ]
+
+    overlays_to_load = [
+        # serial ports, checked individually
+        "uart0-pi5",  # serial1
+        "uart3-pi5",  # serial4
+        "uart4-pi5",  # serial5
+        "uart2-pi5",  # serial3
+        # i2c-6: bar30 and friends
+        "i2c-gpio  i2c_gpio_sda=22 i2c_gpio_scl=23 bus=6 i2c_gpio_delay_us=0",
+        # i2c1: ADS1115, AK09915, BME280
+        "i2c1-pi5  baudrate=400000",
+        # i2c3: PCA
+        "i2c3-pi5  baudrate=400000",
+        # SPI1: MMC5983
+        "spi1-3cs",
+        # SPI0: LED
+        "spi0-led",
+    ]
+
+    for dtparam in dtparams_to_load:
+        if dtparam in loaded_dt_params:
+            logger.info(f"dtparam {dtparam} is already loaded")
+        else:
+            logger.info(f"dtparam {dtparam} is not loaded, loading")
+            logger.info(run_command(f"sudo dtparam {dtparam}", False))
```

Comment:
consider the following for better readability, update the other calls if accepted

```suggestion
            command_result = run_command(f"sudo dtparam {dtparam}", False)
            logger.info(command_result)
```

Reply:
hmmm mixed feelings
we are already using the `logger.info(run_command())` structure at least in 4 other insteances in this file.

We could switch all of them to having an intermediate variable, but I wouldn't do it in this pr. I can fix this  up and do a follow up to update the others later, though. thoughts @patrickelectric  @JoaoMario109 ?

Reply:
actually, I do think readability wins. I'll update the patch